sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -87,19 +82,103 @@ public PollerFlux(Duration defaultPollInterval,
                 "Negative or zero value for 'defaultPollInterval' is not allowed."));
         }
         this.defaultPollInterval = defaultPollInterval;
-        this.activationOperation = Objects.requireNonNull(activationOperation,
-            "'activationOperation' cannot be null.");
-        this.oneTimeActivationMono = oneTimeActivationMono(activationOperation);
+        Objects.requireNonNull(activationOperation, "'activationOperation' cannot be null.");
         this.pollOperation = Objects.requireNonNull(pollOperation, "'pollOperation' cannot be null.");
         this.cancelOperation = Objects.requireNonNull(cancelOperation, "'cancelOperation' cannot be null.");
         this.fetchResultOperation = Objects.requireNonNull(fetchResultOperation,
             "'fetchResultOperation' cannot be null.");
+        this.oneTimeActivationMono = new OneTimeActivation<>(this.rootContext,
+            activationOperation,
+            // mapper
+            activationResult -> new PollResponse<>(LongRunningOperationStatus.NOT_STARTED, activationResult)).getMono();
+        this.syncActivationOperation =
+            cxt -> new PollResponse<>(LongRunningOperationStatus.NOT_STARTED, activationOperation.apply(cxt).block());
+    }
+
+    /**
+     * Creates PollerFlux.
+     *
+     * PollerFlux obtained from this factory method uses an activationOperation which returns a Mono that
+     * emits {@link PollResponse}, which holds the result.
+     * The PollerFlux created from constructor uses an activationOperation which returns a Mono that directly
+     * emits result.
```

Comment:
If you're meaning to contrast this method against the constructor, be more explicit. "This create method differs from the PollerFlux constructor in that the constructor uses an ...."